main.nf

```diff
@@ -302,15 +355,53 @@ def helpMessage() {
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/pangenome --input 'data/*.fa.gz' -profile docker
+    nextflow run nf-core/pangenome --input 'data/input.fa.gz' -profile docker
 
     Mandatory arguments:
-      --input [file]                  Path to input data (must be surrounded with quotes)
+      --input [file]                  Path to input FASTA (must be surrounded with quotes)
       -profile [str]                  Configuration profile to use. Can use multiple (comma separated)
                                       Available: conda, docker, singularity, test, awsbatch, <institute> and more
+    Alignment options:
+      --edyeet_align_pct_id [n]       percent identity in the edyeet edlib alignment step [default: 90]
+      --alignment_map_pct_id [n]      percent identity in the wfmash or edyeet mashmap step [default: 90]
+      --alignment_n_secondary [n]     number of secondary mappings to retain in 'map' filter mode [default: 10]
+      --alignment_segment_length [n]  segment length for mapping [default: 10000]
+      --alignment_block_length [n]    minimum block length filter for mapping [default: 3*alignment_segment_length]
+      --alignment_mash_kmer [n]       kmer size for mashmap [default: 16]
+      --alignment_merge_segments      merge successive mappings [default: OFF]
+      --alignment_no_splits           disable splitting of input sequences during mapping [default: OFF]
+      --alignment_exclude--delim [c]  skip mappings between sequences with the same name prefix before
+                                      the given delimiter character [default: all-vs-all and !self]
+    Seqwish options:
+      --seqwish_min_match_length [n]  ignore exact matches below this length [default: 19]
+      --seqwish_transclose_batch [n]  number of bp to use for transitive closure batch [default: 1000000]
+
+    Smoothxg options:
+      --smoothxg_max_block_weight [n] maximum seed sequence in block [default: 10000]
+      --smoothxg_max_path_jump [n]    maximum path jump to include in block [default: 5000]
+      --smoothxg_max_edge_jump [n]    maximum edge jump before breaking [default: 5000]
+      --smoothxg_max_popa_length [n]  maximum sequence length to put into POA [default: 10000]
+      --smoothxg_consensus_spec [str] consensus graph specification: write the consensus graph to
```

Comment:
Can all of these metadata be written out to something other than the file name?  The PAF prefix is already bad enough.

The GFA specification allows for SAM-style `TAG:TYPE:VALUE` fields in the header `H` line
http://gfa-spec.github.io/GFA-spec/GFA1.html#optional-fields-1

Reply:
I understand that such large file names can be confusing. Putting this information into the file itself is a good idea, at least for GFA. But I don't think it's a sustainable way to do this because when processing GFA, the tools involved here usually switch to the `odgi` format in processing, which doesn't keep track of such information. In the end, I want all the metadata to be available in either an extra file or ideally in the MultiQC report. The same goes for the software versions.

When sharing the results of the pipeline, the actual parameters used are important to understand the results. That's why we put the metadata into the file name.
I will change this behaviour to the following, assuming the input file name was `seq.fa.gz`:
- `DEFAULT`: File names will be `seq.fa.gz.seqwish.gfa`, `seq.fa.gz.smooth.gfa`, or `seq.fa.gz.smooth.consensus@10.gfa`
- `--file_name_prefix`: `prefix.seqwish.gfa`, `prefix.smooth.gfa`, or `prefix.smooth.consensus@10.gfa`. If this argument equals `pggb`, then, and only then, I will write all metadata into the file names, as is the default now.

Would that be fine @heuermh @Zethson @andreaguarracino?

Reply:
This is what we agreed on yesterday, so it's okay with me, if the nf-core world is happy too. Maybe not clean, but functional enough.

Reply:
> When sharing the results of the pipeline, the actual parameters used are important to understand the results. That's why we put the metadata into the file name.

I don't think that metadata should be part of the file name, but rather in the file itself or in an accompanying file.
But if you're only referring to seqwish vs smooth vs smooth.censensus then this is fine I guess.

> --file_name_prefix: prefix.pggb.seqwish.gfa, prefix.pggb.smooth.gfa, or prefix.pggb.smooth.consensus@10.gfa. If this argument equals pggb, then, and only then, I will write all metadata into the file names, as is the default now.

Why writing this stuff into the file name? Just write it into the file or pick it up with MultiQC (which tracks software versions & Nextflow run arguments). 

> But I don't think it's a sustainable way to do this because when processing GFA, the tools involved here usually switch to the odgi format in processing, which doesn't keep track of such information. 

Does the odgi format not support any kinds of comments? Using file names for meta data is just not something that is robust and people do.

> In the end, I want all the metadata to be available in either an extra file or ideally in the MultiQC report. The same goes for the software versions.

So let's start here? I suggest that we don't use the prefix but directly go into the solution that you want to eventually go for anyways. Extra file and MultiQC reports scale. File names don't.

My 2cts :)

Reply:
I updated the file naming scheme accordingly.